src/common/message.ts

```diff
@@ -6,7 +6,7 @@ export interface Message {
     payload?: any;
 }
 
-export type InterpreterMessage = Message & { tabId?: number };
+export type InterpreterMessage = Message & { tabId?: any };
```

Comment:
Keep the type to number and made related changes in other files

Reply:
Made the suggested change